### PR TITLE
Vm desk entitlement

### PR DESCRIPTION
**Pull request by cptera**
Use entitlement service for desktop entitlements

### CODE REVIEW

File: frontend/new-admin-vm/Install-CAM.ps1 [hunk 1]
```diff
@@ -1557,7 +1559,57 @@ brokerLocale=en_US
...
+						# Get the machineId
+						if( ($registerMachineResult.code -eq 409) -and ($registerMachineResult.data.reason.ToLower().Contains("already exist")) ) {
+							# Deployment is already registered so the deplymentId needs to be retrieved
+							$registeredMachine = ""
+							try {
+								$registeredMachine = Invoke-RestMethod -Method Get -Uri ($camSaasBaseUri + "/api/v1/machines") -Body $machineRequest -Headers $tokenHeader
+								$machineId = $registeredMachine.data.machineId
+							} catch {
+								if ($_.ErrorDetails.Message) {
+									$registeredMachine = ConvertFrom-Json $_.ErrorDetails.Message
```
Comment (bill-dall) [inline frontend/new-admin-vm/Install-CAM.ps1]: Are there any side effects from converting from and to JSON? Good or bad?

File: frontend/new-admin-vm/Install-CAM.ps1 [hunk 2]
```diff
@@ -1557,7 +1559,57 @@ brokerLocale=en_US
...
+								}								
+							}
+						} else {
+							$machineId = $registerMachineResult.data.machineId
+						}
 						Write-Host "Machine has been registered succesfully with Cloud Access Manager"
+						
+						# Register User Entitlement to Machine
+						# Get User Guid for Domain User
+						$DCSession = New-PSSession $using:dcvmfqdn -Credential $using:DomainAdminCreds
```
Comment (bill-dall) [inline frontend/new-admin-vm/Install-CAM.ps1]: localhost didn't work? I'm okay wither way just wanted to know if you tried it.

Comment (cptera) [reply frontend/new-admin-vm/Install-CAM.ps1]: I can try - I just copied what we were doing above

Comment (bill-dall) [reply frontend/new-admin-vm/Install-CAM.ps1]: No I was just wondering if you had tried. I want to eventually reduce the amount of remote powershelling, but we can cross that bridge later. Not a good experiment for the last day in a sprint.

**Pull request merged**